python_modules/dagster/dagster/core/execution.py

```diff
@@ -37,13 +37,18 @@
     SystemPipelineExecutionContext,
 )
 
-
 from .errors import DagsterInvariantViolationError
 
 from .events import construct_event_logger
 
 from .execution_plan.create import create_execution_plan_core
 
+from .execution_plan.intermediates_manager import (
+    FileSystemIntermediateManager,
+    InMemoryIntermediatesManager,
```

Comment:
`Intermediates` or `Intermediate`

Reply:
my pluralization, capitalization, and spelling skills continue to plague our codebase